src/settings-tab.ts

```diff
@@ -133,7 +133,7 @@ export class CustomFramesSettingTab extends PluginSettingTab {
                         await this.plugin.saveSettings();
                     });
                 });
-            new Setting(content)
+                new Setting(content)
```

Comment:
nbd but you're doing weird indentation here

Reply:
ngl not sure what happened there but fixed